content/departments/engineering/teams/growth/cloud-trial-operations.md

```diff
@@ -1,50 +1,169 @@
+
+1. `bob` is automatically and instantly redirected to a pre-provisioned instance and given a password reset link. That instance has a random domain (eg. `xyz.sourcegraph.com`). From now on, all `@acme-corp.com` emails will be redirected to that instance (see Case 1).
+1. An alert is sent in #cloud-trial-alerts, powered by this [zap](https://zapier.com/editor/167443639/published)
+1. 🟡 Inbound SDR converts to Opp
+1. 🟡 Inbound SDR uses Round Robin to assign AE/CE
+1. 🟡 Inbound SDR notifies assigned AE/CE
+   1. Auto-email from Salesforce upon Conversion
+   2. [INTERNAL] 🟡 inbound SDR notifies assigned AE/CE in the #cloud-trial-alerts, and creates a slack channel #cloud-trial-<acme-corp> and adds CE + AE.
+1. 🟢 The assigned CE takes over from here, go to [Second step: onboarding, and trial extend, terminate, convert](#second-step-onboarding-and-trial-extend-terminate-convert)
+
+#### Case 3: acme-corp.com is NOT pre-qualified
+
+In that case, `bob@acme-corp.com` does NOT instantly and automatically get a pre-provisioned instance. We need to request an instance for them and give them a reset password link. It also means that the account is NOT a named account in SFDC because all named accounts are prequalified.
+
+There are two cases. It all starts with an alert is sent in #cloud-trial-alerts, powered by this [zap](https://zapier.com/editor/167443639/published) asking Inbound SDR to triage the lead. Then:
+
+##### Case 3.1: the lead is not qualified
+
+1. 🟡 inbound SDR disqualifies.
+1. the user gets an automated email recommending to self-host Sourcegraph
+
+##### Case 3.2: the lead is qualified
+
+1. 🟡 Inbound SDR converts to Opp
+1. 🟡 Inbound SDR uses Round Robin to assign AE/CE
+1. 🟡 Inbound SDR notifies assigned AE/CE
+   1. Auto-email from Salesforce upon Conversion
+   2. 🟡 Inbound SDR notifies assigned AE/CE in the #cloud-trial-alerts, and creates a slack channel #cloud-trial-<acme-corp> and adds CE + AE.
+1. An instance request GitHub issue is raised automatically and posted in channel. This is powered by this [zap](https://zapier.com/editor/173098650/published).
+1. 🟡 Inbound SDR adds the assigned CE email in the GitHub issue. This is very important: otherwise the CE will not be able to access the instance to generate `bob`'s initial password reset link. There is a `TODO` field in the GitHub issue that just needs to be replaced by the assigned CE's Sourcegraph email.
+1. 🟡 Inbound SDR sends a “welcome” email, letting the prospect know that their instance is being provisioned, and introducing the CE who will be able to help with questions (AE in cc).
+1. The Cloud team is paged, provisions cloud instance. A default, generic license key will be automatically added (shared by all trial instances in a cohort). This key is owned and rotated by Malo Marrec every 7 days. This license key has tags `plan:enterprise-1`,`private-extension-registry`,`remote-extensions-allow-disallow`,`monitoring`,`true-up`, `trial`, `plg-trial` and 1,000 users.
```

Comment:
As this is not using pre-provsioned instances (?), this would fall into 1 working day SLA? (same as for any trials)

Reply:
Yes. Does that work?